src/utils/dappMailer.js

```diff
@@ -288,7 +288,7 @@ module.exports = {
         </p>
       `,
       cta: `Manage Milestone`,
-      ctaRelativeUrl: `/my-milestones`,
+      ctaRelativeUrl: `/campaigns/${data.campaignId}/milestones/${data.milestoneId}`,
```

Comment:
`campaignId` and `milestoneId` are not passed to `milestoneMarkedCompleted` function.

Reply:
You are right. I make a new commit, please check.